### PR TITLE
feat(): Update main README + Add svgo in usable libraries

### PR DESCRIPTION
**Pull request by ChucKN0risK**
## What it does

Update main README + Add svgo in usable libraries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,29 +23,32 @@ Parsers are ordered and takes specific input to generate specific output. This w
 
 By using parsers, you dictate the way you receive the data from Specify to fit your own needs.
 
-## Creating your own parser
+## How to create your own parser?
+
+Let's say you want to create a parser named `my-parser`.
 
 1. Git clone this repo
```
Comment (AMoreaux) [inline README.md:30]: Add the fork step at first.
1. Fork the current repository
....

**Review by AMoreaux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ChucKN0risK) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -23,29 +23,33 @@ Parsers are ordered and takes specific input to generate specific output. This w
...
+3. In the directory `parsers`, create a directory named `my-parser`
+4. Create your valid parser
+5. Make a PR
 
 ### Creating a valid parser
 
-To be valid, a parser needs:
-- A `README.md` file which contains:
+To be valid, your parser needs:
+- A `README.md` file which containing:
```
Comment (AllanMichay) [inline README.md:39]: `containing` or `which contains`, not both

Comment (ChucKN0risK) [reply README.md]: DOne

**Review by AMoreaux — APPROVED** [review]

**Review by AMoreaux — APPROVED** [review]

**Pull request merged**